mysql-test/t/backup_locks.test

```diff
@@ -5,6 +5,7 @@
 --source include/have_myisam.inc
 --source include/have_archive.inc
 --source include/have_blackhole.inc
+--source include/have_rocksdb.inc
```

Comment:
I don't agree with this change. Lots of folks build server without MyRocks, and now they would get test disabled. Also, this is not the only one relevant backup locks test, IIRC there was rpl_backup_locks too at least, needs grepping. Please clone into a MyRocks suite with a TODO to work out the currently non-obvious details and convert it to .inc scheme